diff --git a/node.js b/node.js
index 8eb6b650421ede47f87da385710235658875ffca..2cec7a04e30e9425d4aa855e25583519f60210b1 100644
--- a/node.js
+++ b/node.js
@@ -370,21 +370,7 @@ module.exports = {
   },
 
   oldDataWarning: function oldDataWarning(agentsObj) {
-    if (dataTimeChecked) return
-    dataTimeChecked = true
-    if (process.env.BROWSERSLIST_IGNORE_OLD_DATA) return
-
-    var latest = latestReleaseTime(agentsObj)
-    var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
-
-    if (latest !== 0 && latest < halfYearAgo) {
-      console.warn(
-        'Browserslist: caniuse-lite is outdated. Please run:\n' +
-          '  npx browserslist@latest --update-db\n' +
-          '  Why you should do it regularly: ' +
-          'https://github.com/browserslist/browserslist#browsers-data-updating'
-      )
-    }
+    return
   },
 
   currentNode: function currentNode() {
